src/languages/Python.js

```diff
@@ -61,7 +61,7 @@ define(function Python(require, exports, module) {
      */
     function getOutlineList(text) {
         var lines = text.split("\n");
-        var regex = /^([ \t]*)(class|def) +(\w+) *(\([\w, =*.]*\))?:$/g;
+        var regex = /^([ \t]*)(class|def) +(\w+) *(\([\w, =*.]*\))(: *$|: *#+[\w\W]*$)/g;
```

Comment:
1. I think we still need the `?` because the arguments are optional for classes.
2. I don't think we need the group as there is no alternative, just an optional comment.
3. Isn't `[\w\W]*` the same as `.*`?
4. `#` is included in `.` and `\W`, so there is no need for the `+`.

What do you think about this instead:
`?:[ \t]*(?:#.*)?$`

Reply:
Much, much better. It's clear that I'm not an expert on regular expressions, hehe.

I think it is better to close this PR and you make the change. Do you think it's OK?

Reply:
That's fine. I really appreciate your effort. 👍 
You can leave it open, I'll take care of it later.